easybuild/easyconfigs/a/astro-tulips/astro-tulips-1.0.1-foss-2022a.eb

```diff
@@ -0,0 +1,52 @@
+# Astro-tulips
+
+easyblock = 'PythonBundle'
+
+name = 'astro-tulips'
+version = '1.0.1'
+
+homepage = 'https://astro-tulips.readthedocs.io/en/latest/installation.html'
+description = """tulips creates diagrams of the structure and evolution of stars.
+It creates plots and movies based on output from the MESA stellar evolution code"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('matplotlib', '3.5.2'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
+    ('astropy', '5.1.1'),
+    ('IPython', '8.5.0'),
+    ('FFmpeg', '5.0.1'),
+]
+
+
+exts_list = [
+    ('mesaPlot', '1.1.0', {
+        'modulename': 'mesaPlot',
+        'checksums': ['25939830eb94ba192e8927175fa9524ed56645e596c96532c844c951cc4d6153'],
+    }),
+    ('colorspacious', '1.1.2', {
+        'checksums': ['5e9072e8cdca889dac445c35c9362a22ccf758e97b00b79ff0d5a7ba3e11b618'],
+    }),
+    ('cmasher', '1.6.3', {
+        'checksums': ['242e206e58ad6115ff0a5a5405a1ddae80efd1cb03a06dc23db44e710e7c9f23'],
+    }),
+    ('ipywidgets', '8.0.4', {
+        'checksums': ['c0005a77a47d77889cafed892b58e33b4a2a96712154404c6548ec22272811ea'],
+    }),
+    ('imageio-ffmpeg', '0.4.8', {
+        'checksums': ['fdaa05ad10fe070b7fa8e5f615cb0d28f3b9b791d00af6d2a11e694158d10aa9'],
+    }),
+    (name, version, {
+        'modulename': 'tulips',
+        'sources': ['astro_tulips-%(version)s-py3-none-any.whl'],
+        'checksums': ['c13ec2a3048b51f108b9470cd7c3f73d0bc9f908b46492ad07115903b6fc1945'],
+    }),
+]
+
+use_pip = True
```

Comment:
Some things are still missing here, e.g. `sanity_pip_check = True`, a `sanity_check_paths` and a `moduleclass`.